### PR TITLE
renamed SFC in the more common phrase GND and moved it to language file.

### PR DESCRIPTION
**Pull request by AlphaLima**
just a small change for a very old cause of misunderstandings.

### CODE REVIEW

File: Common/Source/Airspace/LKAirspace.cpp [hunk 1]
```diff
@@ -3216,7 +3216,7 @@ void CAirspaceManager::GetAirspaceAltText(TCHAR *buffer, int bufferlen, const AI
             break;
         case abAGL:
             if (alt->AGL <= 0)
-                _stprintf(intbuf, TEXT("SFC"));
+                _stprintf(intbuf, MsgToken(2387)); //_@M2387_ "GND"
```
Comment (brunotl) [inline Common/Source/Airspace/LKAirspace.cpp]: sprintf with user format string : unsafe

File: Common/Source/Airspace/LKAirspace.cpp [hunk 2]
```diff
@@ -3254,7 +3254,7 @@ void CAirspaceManager::GetSimpleAirspaceAltText(TCHAR *buffer, int bufferlen, co
             break;
         case abAGL:
             if (alt->AGL <= 0)
-                _stprintf(intbuf, TEXT("SFC"));
+                _stprintf(intbuf, MsgToken(2387)); // _@M2387_ "GND"
```
Comment (brunotl) [inline Common/Source/Airspace/LKAirspace.cpp]: sprintf with user format string : unsafe

File: Common/Source/Airspace/LKAirspace.cpp (hunk 1 above)
Comment (AlphaLima) [reply Common/Source/Airspace/LKAirspace.cpp]: solved

File: Common/Source/Airspace/LKAirspace.cpp (hunk 2 above)
Comment (AlphaLima) [reply Common/Source/Airspace/LKAirspace.cpp]: solved

**Comment by AlphaLima** [comment]
done!

**Pull request merged**